src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/MSBuildProjectManager.cs

```diff
@@ -193,7 +193,7 @@ private void UpdateProjectState(ProjectInstance projectInstance)
         foreach (var documentFilePath in projectSnapshot.DocumentFilePaths)
         {
             OmniSharpHostDocument? associatedHostDocument = null;
-            var currentHostDocument = projectSnapshot.GetDocument(documentFilePath).HostDocument;
+            var currentHostDocument = projectSnapshot.GetDocument(documentFilePath)!.HostDocument;
```

Comment:
I'm wondering if instead of using the dreaded `!` we should actually be doing something like:
> if (document is null){ Debug.Fail("Unable to find a document we just got from the project"); }

Reply:
That would change runtime behaviour, so no. I'd be fine with `Assumes.NotNull` before this line though.